controls/ism_o.yml

```diff
@@ -0,0 +1,724 @@
+            - disable_host_auth
+            - require_emergency_target_auth
+            - require_singleuser_auth
+            - sebool_authlogin_nsswitch_use_ldap
+            - sebool_authlogin_radius
+            - sshd_disable_kerb_auth
+            - sshd_set_max_auth_tries
+            - sssd_enable_smartcards
+        status: automated
+
+    -   id: '1559'
+        title: 'Memorised secrets used for multi-factor authentication are a minimum of 6 characters, unless more stringent requirements apply.'
+        levels:
+            - base
+        rules:
+            - accounts_password_minlen_login_defs
+            - accounts_password_pam_dcredit
+            - accounts_password_pam_lcredit
+            - accounts_password_pam_minclass
+            - accounts_password_pam_minlen
+            - accounts_password_pam_ocredit
+            - accounts_password_pam_ucredit
+            - accounts_passwords_pam_faillock_deny
+            - accounts_passwords_pam_faillock_deny_root
+            - accounts_passwords_pam_faillock_interval
+            - accounts_passwords_pam_faillock_unlock_time
+            - accounts_passwords_pam_tally2_deny_root
+            - accounts_passwords_pam_tally2_unlock_time
+            - disable_host_auth
+            - require_emergency_target_auth
+            - require_singleuser_auth
+            - sebool_authlogin_nsswitch_use_ldap
+            - sebool_authlogin_radius
+            - sshd_disable_kerb_auth
+            - sshd_set_max_auth_tries
+            - sssd_enable_smartcards
+        status: automated
+
+    -   id: '1560'
+        title: 'Memorised secrets used for multi-factor authentication on SECRET systems are a minimum of 8 characters'
+        levels:
+            - secret
+        rules:
+            - accounts_password_minlen_login_defs
+            - accounts_password_pam_dcredit
+            - accounts_password_pam_lcredit
+            - accounts_password_pam_minclass
+            - accounts_password_pam_minlen
+            - accounts_password_pam_ocredit
+            - accounts_password_pam_ucredit
+            - accounts_passwords_pam_faillock_deny
+            - accounts_passwords_pam_faillock_deny_root
+            - accounts_passwords_pam_faillock_interval
+            - accounts_passwords_pam_faillock_unlock_time
+            - accounts_passwords_pam_tally2_deny_root
+            - accounts_passwords_pam_tally2_unlock_time
+            - disable_host_auth
+            - require_emergency_target_auth
+            - require_singleuser_auth
+            - sebool_authlogin_nsswitch_use_ldap
+            - sebool_authlogin_radius
+            - sshd_disable_kerb_auth
+            - sshd_set_max_auth_tries
+            - sssd_enable_smartcards
+        status: automated
+
+    -   id: '1561'
+        title: 'Memorised secrets used for multi-factor authentication on TOP SECRET systems are a minimum of 10 characters.'
+        levels:
+            - top_secret
```

Comment:
Shouldn't there be multiple profiles that correspond to the different levels of the ISM?

At least the profile description should mention which level it corresponds to.